Sources/Fuzzilli/FuzzIL/Context.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Current context in the program
+public struct Context: OptionSet {
+    public let rawValue: Int
+    
+    public init(rawValue: Int) {
+        self.rawValue = rawValue
+    }
+    
+    // Default script context
+    public static let script            = Context([])
```

Comment:
This now needs to have a value (i.e. `Context(rawValue: 1 << 0)`), because otherwise the subset tests won't work correctly (the `script` context would be a subset of every possible context since it's the empty set)

Reply:
And yeah, I think we can just keep calling it `.script` then :)